templates/project_puppeteer_crawler/main.js

```diff
@@ -26,6 +26,11 @@ Apify.main(async () => {
             useChrome: true,
             stealth: true,
```

Comment:
I thought fingerprints would completely replace `stealth`. How do those two work together? Should we add `stealth` to Playwright as well?

Reply:
Stealth masks headless chrome to look like not automated chrome by flipping a few navigator properties. Fingerprints change already existing properties to provide higher entropy in the target browser.

Reply:
Thanks, but the question still stands: Do we need stealth for Playwright? Also, do we need to update the existing stealth as it's pretty outdated?

Reply:
We should add the argument behind `hideWebDriver` to the default launch arguments for chrome/chromium and then deprecate the stealth completely.